<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePcodeDialog</class>
 <widget class="QWidget" name="CreatePcodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>364</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use this form to create a Receiver Address Privacy (BIP47) payment code.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>An optional label to associate with the new receiving address.</string>
          </property>
          <property name="text">
           <string>&amp;Label:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>reqLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="createPcodeButton">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>C&amp;reate payment code</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../res/icons/paymentcode.png</normaloff>../res/icons/paymentcode.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Clear all fields of the form.</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="reqLabel">
          <property name="toolTip">
           <string>An optional label to associate with the new receiving address.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recent payment codes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="recentRequestsView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="showPcodeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Show the selected request (does the same as double clicking an entry)</string>
          </property>
          <property name="text">
           <string>Show</string>
          </property>
          <property name="icon">
           <iconset resource="../bitcoin.qrc">
            <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reqLabel</tabstop>
  <tabstop>createPcodeButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>recentRequestsView</tabstop>
  <tabstop>showPcodeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
